pkg/controller/account/account_controller_test.go

```diff
@@ -1056,3 +1060,94 @@ func TestGetAssumeRole(t *testing.T) {
 		)
 	}
 }
+
+var _ = Describe("Account Controller", func() {
+	var (
+		nullLogger    testutils.NullLogger
+		mockAWSClient *mock.MockClient
+		accountName   string
+		accountEmail  string
+		ctrl          *gomock.Controller
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockAWSClient = mock.NewMockClient(ctrl)
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	Context("Testing CreateAccount", func() {
+		It("AWS returns an error from CreateAccount", func() {
+			// ErrCodeServiceException is mapped to awsv1alpha1.ErrAwsInternalFailure in CreateAccount
+			mockAWSClient.EXPECT().CreateAccount(gomock.Any()).Return(nil, awserr.New(organizations.ErrCodeServiceException, "Error String", nil))
+			createAccountOutput, err := CreateAccount(nullLogger, mockAWSClient, accountName, accountEmail)
+			Expect(err).To(HaveOccurred())
+			Expect(createAccountOutput).To(Equal(&organizations.DescribeCreateAccountStatusOutput{}))
+			Expect(awsv1alpha1.ErrAwsInternalFailure).To(Equal(err))
+		})
+
```

Comment:
Could test return for ErrAwsTooManyRequests and ErrAwsFailedCreateAccount too to increase the coverage

Reply:
Added